test_scripts/API/ATF_GetSystemCapability.lua

```diff
@@ -75,6 +75,36 @@ function Test:TestStep_GetPhoneCapability()
   :Timeout(12000)   
 end
 
+function Test:TestStep_GetVideoCapability()
+  local CorIdGetSystemCapability = self.mobileSession:SendRPC(
+    "GetSystemCapability",
+    {
+      systemCapabilityType = 2
```

Comment:
Probably should send a string rather than the direct enum value here.

Reply:
Tests were using int/enum value. I believe this is ok

Reply:
I disagree, theoretically these values could change in the future. They are not defined in the API, just internally by Core